tests/system/common.py

```diff
@@ -737,3 +737,13 @@ def marathon_leadership_changed(original_leader):
     """
     __marathon_leadership_changed_in_marathon_api(original_leader)
     __marathon_leadership_changed_in_mesosDNS(original_leader)
+
+
+def running_status_network_info(task_statuses):
+    """ From a given list of statuses retrieved from mesos API it returns network info of running task.
+    """
+    for task_status in task_statuses:
+        if task_status['state'] == "TASK_RUNNING":
+            return task_status['container_status']['network_infos'][0]
```

Comment:
Somehow it seems to me that we make a lot of assumptions here. E.g. we retrieve just the first network info of the first task that is task_running. Shouldn't identify the task by name?

Reply:
I am querying tasks by name now, good point